composer.json

```diff
@@ -83,7 +83,8 @@
 		"exclude-from-classmap": [
 			"./Services/Migration",
 			"./*/*/lib", "./*/*/test",
-			"./Services/CAS/phpcas"
+			"./Services/CAS/phpcas",
+			"./Customizing/**/vendor"
```

Comment:
Hi @theodortruffer,

just for curiosity: Is there any reason that there are two `**` here?

Best regards!

Reply:
Hey Richard

Single Asterisk matches everything but a slash and double asterisk everything, including slashes. So * is just one directory, ** can be a whole subpath. See https://getcomposer.org/doc/04-schema.md#exclude-files-from-classmaps